src/plugins/WebIndexPlugin.ts

```diff
@@ -86,7 +87,7 @@ $bundles
 
         let jsTags = this.opts.emitBundles
             ? this.opts.emitBundles(bundlePaths)
-            : bundlePaths.map(bundle => `<script ${this.opts.async ? 'async' : ''} type="text/javascript" src="${bundle}"></script>`).join('\n');
+            : bundlePaths.map(bundle => `<script ${this.opts.async ? 'async' : ''} ${this.opts.defer ? 'defer' : ''} type="text/javascript" src="${bundle}"></script>`).join('\n');
```

Comment:
I think you should check if there is async or defer to avoid adding async plus defer and vice versa.

Reply:
Ok, I got your point! Would you like me to keep both attributes as properties of the WebIndexPlugin or should I rewrite the as a generic js_additional_script_attributes (of type string)?

Reply:
I think in that case, making `js_additional_script_attributes` would be more better.
It' d allow to do more things like for example add an `ID`